openshift/go.mod

```diff
@@ -0,0 +1,5 @@
+module _
+
+go 1.19
```

Comment:
Don't forget to bump to 1.20 here so we maintain alignment between up/down-stream.